orttraining/orttraining/models/bert/main.cc

```diff
@@ -136,6 +136,11 @@ Status ParseArguments(int argc, char* argv[], BertParameters& params, OrtParamet
       ("beta", "Adam/Lamb beta parameter", cxxopts::value<float>()->default_value("0.999"))
       ("lambda", "Adam/Lamb lambda parameter", cxxopts::value<float>()->default_value("0.01"))
       ("epsilon", "Adam/Lamb epsilon parameter", cxxopts::value<float>()->default_value("1e-6"))
+      ("do_bias_correction",
+        "A flag controls if Adam/Lamb should do bias correction. "
+        "Default is false, which means no bias correction. "
+        "Use true to enable bias correction.",
+        cxxopts::value<bool>()->default_value("false"))
```

Comment:
>false [](start = 47, length = 5)

the schema is default 1 

shall they match ?

Reply:
I feel it's better not to change the behavior of C++ frontend. It'd be surprise if the same command produces very different results.